src/ui/preferences/GeneralSettings.qml

```diff
@@ -982,6 +982,98 @@ Rectangle {
                         }
                     }
 
+                    Item { width: 1; height: _margins; visible: ntripSectionLabel.visible }
+                    QGCLabel {
+                        id:         ntripSectionLabel
+                        text:       qsTr("NTRIP / RTCM")
+                        visible:    QGroundControl.settingsManager.ntripSettings.visible
+                    }
+                    Rectangle {
+                        Layout.preferredHeight: ntripGrid.y + ntripGrid.height + _margins
```

Comment:
[Layout.margins](https://doc.qt.io/qt-5/qml-qtquick-layouts-layout.html#margins-attached-prop) ?
and Layout.topMargin